client/dashboard/sites/settings/index.tsx

```diff
@@ -36,6 +37,7 @@ export default function SiteSettings( { siteSlug }: { siteSlug: string } ) {
 				<DatabaseSettingsSummary site={ site } />
 				<WordPressSettingsSummary site={ site } />
 				<PHPSettingsSummary site={ site } />
+				<AgencyFullyManagedSummary site={ site } />
```

Comment:
I know it's from v1, but I feel the naming sounds kinda convoluted. Could it be just `AgencySettingsSummary`, `settings-agency/` directory, `settings/agency` path? The "fully managed" part is just a field inside the settings. 🤔